tools/scylla-nodetool.cc

```diff
@@ -3709,6 +3718,7 @@ For more information, see: {}"
                             typed_option<std::vector<sstring>>("in-dc", "Constrain repair to specific datacenter(s)"),
                             typed_option<std::vector<sstring>>("in-hosts", "Constrain repair to the specific host(s)"),
                             typed_option<std::vector<sstring>>("tablet-tokens", "Tokens owned by the tablets to repair."),
+                            typed_option<sstring>("incremental-mode", "Specify the incremental repair mode"),
```

Comment:
I think this at least should list valid modes, as a quick reminder for somebody not remembering how to do a full repair. It is ok to not have full explanation on what each mode means, we have online docs for that.

Reply:
OK. Makes sense.